### PR TITLE
Update CI/CD

### PR DESCRIPTION
**Pull request by madil90**

### CODE REVIEW

**Comment by IsaacYangSLA** [comment]
@madil90 , can you help take a look at this PR and ensure there is no conflict?

**Comment by madil90** [comment]
@IsaacYangSLA All issues are resolved.

**Review by YuanTingHsieh — DISMISSED** [review]
LGTM, just one note this run_app_tests.sh does not have newline at end of file.

**Comment by madil90** [comment]
> LGTM, just one note this run_app_tests.sh does not have newline at end of file.

Fixed

**Review by nvkevlu — DISMISSED** [review]
LGTM

**Review by yhwen — APPROVED** [review]
LGTM

File: .github/workflows/premerge.yml [hunk 1]
```diff
@@ -2,14 +2,14 @@ name: pre-merge
 
 on:
   # quick tests for pull requests and the releasing branches
```
Comment (YuanTingHsieh) [inline .github/workflows/premerge.yml:4]: Does this mean runtest.sh / run_example_tests.sh / run_app_tests.sh are all quick tests?

File: .github/workflows/blossom-ci.yml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      #  uses: actions/setup-java@v1
+      #  with:
+      #    java-version: 1.8
+
+      # add blackduck properties https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/631308372/Methods+for+Configuring+Analysis#Using-a-configuration-file
+      #- name: Setup blackduck properties
+      #  run: |
+      #       PROJECTS=$(mvn -am dependency:tree | grep maven-dependency-plugin | awk '{ out="com.nvidia:"$(NF-1);print out }' | grep rapids | xargs | sed -e 's/ /,/g')
+      #       echo detect.maven.build.command="-pl=$PROJECTS -am" >> application.properties
+      #       echo detect.maven.included.scopes=compile >> application.properties
```
Comment (YuanTingHsieh) [inline .github/workflows/blossom-ci.yml:70]: should we delete these comments?

File: .github/workflows/premerge.yml (hunk 1 above)
Comment (madil90) [reply .github/workflows/premerge.yml:4]: No, the change means that we'll need to issue a "/build" comment to run ci now

File: .github/workflows/blossom-ci.yml (hunk 2 above)
Comment (madil90) [reply .github/workflows/blossom-ci.yml:70]: They came from Blossom. There maybe a situation in which we need to use them so I've kept them here for now. We can clean up once integration is working

**Review by nvkevlu — DISMISSED** [review]

**Review by YuanTingHsieh — DISMISSED** [review]
LGTM

**Comment by YuanTingHsieh** [comment]
@madil90 you will need to sign the commits for this to be merged, thanks!

**Comment by madil90** [comment]
Branch rebased and commits signed.

**Review by YuanTingHsieh — APPROVED** [review]

**Review by nvkevlu — APPROVED** [review]

**Pull request closed without merging**